mne/io/bti/bti.py

```diff
@@ -1116,13 +1129,19 @@ def _get_bti_info(pdf_fname, config_fname, head_shape_fname, rotation_x,
     use_hpi = False  # hard coded, but marked as later option.
     logger.info('Creating Neuromag info structure ...')
     info = _empty_info()
-    date = bti_info['processes'][0]['timestamp']
-    info['meas_date'] = [date, 0]
-    info['sfreq'] = 1e3 / bti_info['sample_period'] * 1e-3
+    if pdf_fname:
```

Comment:
funny, we had a discussion on this recently with you, you argued it's a mess that we introduced this explicit tests everywhere, actually convincing me it's more pythonic like this.